### PR TITLE
Improve collected Elasticsearch metrics.

### PR DESCRIPTION
**Pull request by piotrm50**
# Description of change

Add additional remotemetrics for easier analysis.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested
Ran docker network and checked if metrics are collected correctly.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/remotelogmetrics/events.go [hunk 1]
```diff
@@ -63,6 +63,10 @@ type TransactionMetrics struct {
 	ScheduledTimestamp time.Time `json:"scheduledTimestamp" bson:"scheduledTimestamp"`
 	BookedTimestamp    time.Time `json:"bookedTimestamp" bson:"bookedTimestamp"`
 	ConfirmedTimestamp time.Time `json:"confirmedTimestamp" bson:"confirmedTimestamp"`
+	DeltaSolid         int64     `json:"deltaSolid"`
```
Comment (NaitsabesMue) [inline packages/remotelogmetrics/events.go:66]: In Kibana, there is only the host and not the nodeID, but several nodes running on the same host. Could you add the nodeID as information?

**Review by NaitsabesMue — APPROVED** [review]
Add nodeID to transactionMetric

File: plugins/remotelogmetrics/drng.go [hunk 2]
```diff
@@ -19,6 +19,7 @@ func onRandomnessReceived(state *drng.State) {
 		Round:             state.Randomness().Round,
 		IssuedTimestamp:   state.Randomness().Timestamp,
 		ReceivedTimestamp: clock.SyncedTime(),
+		DeltaReceived:     clock.SyncedTime().Sub(state.Randomness().Timestamp).Nanoseconds(),
```
Comment (capossele) [inline plugins/remotelogmetrics/drng.go:22]:
```suggestion
		DeltaReceived:     clock.Since(state.Randomness().Timestamp).Nanoseconds(),
```

Comment (capossele) [reply plugins/remotelogmetrics/drng.go:22]: something like this is more readable

**Pull request merged**